src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -159,6 +159,10 @@ export default class FiatConnectQuote extends NormalizedQuote {
     return resolveCurrency(this.quote.quote.cryptoType)!
   }
 
+  getCicoCryptoType(): CiCoCurrency {
+    return resolveCICOCurrency(this.quote.quote.cryptoType)
+  }
```

Comment:
Can you keep this out of the quote? I'd rather have FiatConnectQuote only deal with one currency type even though its annoying that our codebase has several

Reply:
Will do!